### ISSUE TITLE
Went from getting a false-positive on AWS053, to an error on the same code.

### ISSUE THREAD

**Issue by MrJoy** [tfsec/tfsec#542]
**Describe the bug**

I was getting an apparent false-positive of AWS053 on my repo, that seems to be #506.  I upgraded to tfsec v0.36.10 and now I get:

```
WARNING: skipped AWS053 check due to error(s): interface conversion: hclsyntax.Expression is *hclsyntax.ScopeTraversalExpr, not *hclsyntax.TemplateExpr
```

**To Reproduce**

Create a module with `variables.tf` as:

```terraform
variable "insights_kms_arn" {
  type        = string
  description = "ARN for the KMS key to use for encrypting Performance Insights data."
}
```

And `main.tf` as:

```terraform
resource "aws_db_instance" "something" {
  identifier = "whatever"

  engine                     = "postgres"
  engine_version             = "12.4"

  instance_class = "db.t3.small"

  deletion_protection   = true
  storage_encrypted     = true
  allocated_storage     = 30
  max_allocated_storage = 1000

  multi_az             = false
  db_subnet_group_name = "subnet-whatever"

  name                   = "something"
  username               = "something"
  password               = "something"
  vpc_security_group_ids = ["sg-123456"]

  kms_key_id = var.data_kms_arn

  performance_insights_enabled    = true
  performance_insights_kms_key_id = var.insights_kms_arn
}
```

**Expected behavior**

I expected to either get no AWS053 error, or to get an indication that my code was still somehow in violation of it.

**Comment by owenrumney** [comment]
Hey @MrJoy  - sorry for the delay in responding. I think this was resolved in #540 by adding support for `ScopeTraversalExpr` would you be able to test with > v0.36.11 to confirm its resolve for you?

I've tested using your example and it looks okay now!

**Comment by MrJoy** [comment]
This has indeed resolved my issue.  Thank you so much!
